packages/angular/common/src/providers/menu-controller.ts

```diff
@@ -1,17 +1,23 @@
 import { Injectable } from '@angular/core';
-import { menuController } from '@ionic/core/components';
 
 @Injectable({
   providedIn: 'root',
 })
 export class MenuController {
+  private menuController;
+
+  // TODO Now - Add type
+  constructor(private menuCtrl: any) {
```

Comment:
I think we should remove the `@Injectable` from this class in common. We aren't using it with dependency injection in the extending implementations. If we did based on the current implementation, Angular would not know how to resolve `menuCtrl` and would throw an exception. 

Since we are treating this as a base class of logic that is extended in the hydrated and custom elements implementations, I think we are safe to remove it. 

re: Type, could you import the type from of `menuController` from `@ionic/core/components`? We already have some instances in the common package where we are assuming the CE build exists in the distributed package.

Reply:
Removed Injectable in base class in https://github.com/ionic-team/ionic-framework/pull/28343/commits/fb772c62627bf6c0c236e4328864055e3abba6d5

Re: the menuController type. We discussed offline, but the current controller interface was wrong so I fixed it in https://github.com/ionic-team/ionic-framework/pull/28343/commits/c102863f0267af17ab4451daa1edafaaf213a8f3